src/prerecorded/mod.rs

```diff
@@ -0,0 +1,262 @@
+#![warn(missing_docs)]
+
+//! This module provides various types that are used for pre-recorded requests to Deepgram.
+
+use crate::{Deepgram, DeepgramError};
+use reqwest::RequestBuilder;
+
+mod audio_source;
+mod options;
+mod response;
+
+pub use audio_source::{BufferSource, UrlSource};
+pub use options::{Keyword, Language, Model, Options, OptionsBuilder, Redact};
+pub use response::{
+    CallbackResponse, ChannelResult, Hit, ListenMetadata, ListenResults, Response,
+    ResultAlternative, SearchResults, Utterance, Word,
+};
+
+use audio_source::AudioSource;
+use options::SerializableOptions;
+use serde::de::DeserializeOwned;
+
+static DEEPGRAM_API_URL_LISTEN: &str = "https://api.deepgram.com/v1/listen";
+
+impl<K: AsRef<str>> Deepgram<K> {
+    /// Sends a request to Deepgram to transcribe pre-recorded audio.
+    /// If you wish to use the Callback feature, you should use [`Deepgram::callback_request`] instead.
+    ///
+    /// The `source` parameter is either a [`BufferSource`] or a [`UrlSource`].
+    ///
+    /// See the [Deepgram API Reference](https://developers.deepgram.com/api-reference/#transcription-prerecorded) for more info.
```

Comment:
I'll leave this comment just here, but it applies to all the docs. The link is a little long, and it might format nicer if you extract it into a link reference, like
```
/// See the [Deepgram API Reference](api) for more info.
/// ...
/// [api]: https://developers.deepgram.com/api-reference/#transcription-prerecorded
```